include/exadg/incompressible_navier_stokes/spatial_discretization/operators/viscous_operator.h

```diff
@@ -431,32 +431,14 @@ class ViscousKernel
 
   // clang-format off
   /*
-   *  These two functions calculates the velocity gradient in normal
-   *  direction depending on the operator type, the type of the boundary face
-   *  and the given boundary conditions.
+   *  These two functions calculates the interor velocity gradient in normal
```

Comment:
```suggestion
   *  These two functions calculate the interior velocity gradient in normal
```

Reply:
thanks! I corrected this and even a bit more than this :)